Uplift/Views/Onboarding/CreateProfileView.swift

```diff
@@ -0,0 +1,190 @@
+//
+//  CreateProfileView.swift
+//  Uplift
+//
+//  Created by Belle Hu on 9/22/24.
+//  Copyright © 2024 Cornell AppDev. All rights reserved.
+//
+
+import PhotosUI
+import SwiftUI
+
+struct CreateProfileView: View {
+
+    // MARK: - Properties
+
+    @EnvironmentObject var mainViewModel: MainView.ViewModel
+    @State private var didCheckTerms: Bool = false
+    @State private var didCheckData: Bool = false
+    @State private var didCheckLocation: Bool = false
+    @State private var profileImage: UIImage?
+    @State private var didShowImagePicker = false
+    @State private var profileItem: PhotosPickerItem?
+
+    // MARK: - UI
+
+    var body: some View {
+        VStack {
+            HStack {
+                Text("Complete your profile.")
+                    .font(Constants.Fonts.h1)
+                    .padding(.leading, 16)
+
+                Spacer()
+            }
+
+            DividerLine()
+                .upliftShadow(Constants.Shadows.smallLight)
+                .padding(.bottom, 46)
+
+            cameraPlaceholderButton
+                .padding(.bottom, 24)
```

Comment:
I would add the padding inside the `cameraPlaceholderButton` since we're already abstracting it away